indexer/services/comlink/src/controllers/api/v4/asset-positions-controller.ts

```diff
@@ -52,8 +52,8 @@ const router: express.Router = express.Router();
 const controllerName: string = 'asset-positions-controller';
 
 @Route('assetPositions')
-class AddressesController extends Controller {
-  @Get('/')
+class AssetPositionsController extends Controller {
+  @Get('/:address/:subaccountNumber')
```

Comment:
This should still be `/` because the parameters are query parameters and not path parameters.

Reply:
Those should still be the same?

I only add this because the other api will look like
`/:address/parentSubaccountNumber/:parentSubaccountNumber`

Reply:
If you look at the doc changes, this changes the `v4/assetPositions` documentation to have the URL be `fetch('https://dydx-testnet.imperator.co/v4/assetPositions/{address}/{subaccountNumber}?address=string&subaccountNumber=0',` which I don't think is correct as there are additional path parameters which was not added in this PR.

Reply:
This just allows you to add the args as path parameters. I think you still have the option to specify the args in the payload instead?

Reply:
`'https://dydx-testnet.imperator.co/v4/assetPositions/{address}/{subaccountNumber}?address=string&subaccountNumber=0'` is not a valid URL today. This means the documentation update is wrong right?
We don't want to change any URLs that work today, so I don't think adding the the args as path parameters is correct for the existing endpoint.